### ISSUE TITLE
CI task "docker-worker-chunk-1" through "-4" are failing

### ISSUE THREAD

**Issue by jwhitlock** [taskcluster/taskcluster#5067]
**Describe the bug**

On January 4th, all CI tests passed on PR #5032. On January 7th for PR #5037 the four tests `docker-worker-chunk-1` through `docker-worker-chunk-4` started failing, and have failed since.

**To Reproduce**

Go to a recent PR, such as PR #5062. All tests pass except for the `docker-worker-chunk-*` tests. I have not yet attempted to reproduce locally.

**Expected behavior**

All tests should pass, especially if they don't interact with Docker Worker.

**Taskcluster version**

PRs against current main branch. Version 44.2.2 is the current release.

**Comment by jwhitlock** [comment]
I stumbled on issue #4968 again, which led me to look for the root cause of that again, and there is a similar error:

```
Error: (HTTP code 404) unexpected - manifest for taskcluster/livelog:v44.3.1 not found
```

**Comment by jwhitlock** [comment]
v4.4.0 was (successfully) released, and is on the tags page:

https://hub.docker.com/layers/taskcluster/livelog/v44.4.0/images/sha256-94093b5b971de7e46e5f3526ba3e7e260d567ee32afaab69ec5322819810c251?context=explore

**Comment by jwhitlock** [comment]
Something else broke on release (issue #5070), but this is fixed in PR #5071, and when that was fixed, the docker-worker-chunks all passed.
